osbuild/testutil/net.py

```diff
@@ -24,6 +28,29 @@ def __init__(self, *args, **kwargs):  # pylint: disable=unused-argument
 
 from .atomic import AtomicCounter
 
+# tree prefix components
+space =  '    '
+branch = '│   '
+# pointers:
+tee =    '├── '
+last =   '└── '
+
+
+def tree(dir_path: Path, prefix: str=''):
```

Comment:
Tree is an extremely common word in this repository and it's used to refer to the location where an osbuild stage is running or targeting (the pipeline tree).  Might not be an issue since this is a test module, but also might avoid confusion to call it something else.